shared/src/business/useCases/getCasesForUserInteractor.js

```diff
@@ -1,41 +1,43 @@
 const { CASE_STATUS_TYPES } = require('../entities/EntityConstants');
+const { compareISODateStrings } = require('../utilities/sortFunctions');
 const { UserCase } = require('../entities/UserCase');
 
 /**
- * getOpenConsolidatedCasesInteractor
+ * getCasesForUserInteractor
  *
  * @param {object} applicationContext the application context
  * @returns {object} the open cases data
```

Comment:
Small nit and sorry, should have captured this on the PR to `test`.  `@returns` should be updated because more than the open cases are now being returned.

Reply:
No problem, I really don't mind comments on staging PRs! Sometimes it just happens that we miss the PR to test... Thanks for the review, I'll update that now!